docs/vscode/tasks.json

```diff
@@ -1,42 +1,17 @@
 {
     // See https://code.visualstudio.com/docs/editor/tasks
     // for the documentation about the tasks.json format
+	
     "version": "2.0.0",
     "tasks": [
         {
-            "taskName": "Lobster",
-            "windows": {
-                "command": "bin\\lobster.exe"
-            },
-            "linux": {
-                "command": "bin/lobster"
-            },
-            "osx": {
-                "command": "bin/lobster"
-            },
-            "type": "shell",
-            "args": [ "--runtime-verbose", "--main", "main.lobster", "${file}" ],
-            "presentation": {
-                "reveal": "silent",
-                "echo": true,
-                "panel": "shared",
-                "focus": false
-            },
-            "group": {
-                "kind": "build",
-                "isDefault": true
-            },
-            "problemMatcher": {
-                "owner": "lobster",
-                "fileLocation": [ "relative", "${fileDirname}" ],
-                "pattern": {
-                    "regexp": "^(.*)\\((.+)\\): (warning|error): (.*)$",
-                    "file": 1,
-                    "line": 2,
-                    "severity": 3,
-                    "message": 4
-                }
-            }
+			// A example of a lobster task
+            "label": "Run lobster",
+            "type": "lobster",
+            "main": "main.lobster",
```

Comment:
this doesn't seem correct.. the above code you deleted can run the current file, this would only be able to run `main.lobster` ? Does the extension take care of platform differences?

`tasks.json` is a file people can copy to their project, as opposed to being an extension.. it should probably remain working even if the lobster extension is not installed

Reply:
Oh ok, i thought it was meant as a example on how to use the extension, Il revert that back